firehose.c

```diff
@@ -103,6 +103,8 @@ static void firehose_response_log(xmlNode *node)
 
 	value = xmlGetProp(node, (xmlChar*)"value");
 	printf("LOG: %s\n", value);
+
```

Comment:
un-needed empty line

Reply:
Will be removed.

Reply:
I have updated patch with removed extra line.